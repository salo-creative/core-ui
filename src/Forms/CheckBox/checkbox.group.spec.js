import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../test';

// COMPONENT
import CheckBoxGroup from './checkBox.group';

test('Renders against snapshot', async () => {
  const { container } = renderWithTheme(
    <CheckBoxGroup />
  );
  expect(container.firstChild).toMatchSnapshot();
});