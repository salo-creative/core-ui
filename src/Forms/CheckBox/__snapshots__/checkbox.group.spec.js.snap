// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders against snapshot 1`] = `
.c0 {
  width: 100%;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  margin: 0 0 2rem 0;
}

.c1 {
  padding: 1rem;
  background: #fafcff;
  border: 0.1rem solid;
  border-color: rgba(0,172,237,0.5);
  border-radius: 0.4rem;
}

<div
  class="c0  salo-checkboxGroup"
>
  
  <div
    class="c1 salo-checkboxGroup__wrapper"
  />
</div>
`;
